packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -169,6 +195,13 @@ class AnalysisServer {
       '--sdk',
       sdk,
     ];
+    // Let the analysis server know that the flutter repository is being analyzed
+    // so that it can enable the public_member_api_docs lint even though
+    // the analysis_options file does not have that lint enabled.
```

Comment:
maybe instead of special-casing the flutter repo like this in the server, we should just give a "--dartdocs" option to the analyzer? Or a command-line option to enable/disable specific lints?